pkg/jtracer/empty_test.go

```diff
@@ -14,18 +14,4 @@
 
 package jtracer_test
 
-import (
-	"testing"
-
-	"github.com/opentracing/opentracing-go"
-	"github.com/stretchr/testify/assert"
-
-	"github.com/jaegertracing/jaeger/pkg/jtracer"
-)
-
-func TestOT(t *testing.T) {
-	mockTracer := opentracing.NoopTracer{}
-	jtracer := jtracer.OT(opentracing.NoopTracer{})
-
-	assert.Equal(t, mockTracer, jtracer.OT)
-}
+// TODO
```

Comment:
if you keep this file, you cannot pass the code coverage check in CI
```
codecov/patch — 29.72% of diff hit (target 95.00%)
```

Reply:
Thanks lot for that call.